crates/forge-runner/src/trace_data.rs

```diff
@@ -15,40 +20,42 @@ pub const TRACE_DIR: &str = "snfoundry_trace";
 
 /// Tree structure representing trace of a call.
 #[derive(Debug, Clone, Deserialize, Serialize)]
-pub struct CallTrace {
-    pub entry_point: CallEntryPoint,
-    pub nested_calls: Vec<CallTrace>,
+pub struct ProfilerCallTrace {
```

Comment:
Renamed the structs since without it I cannot comprehend which struct is profiler related and which is not while writing the code

Reply:
Why do we even have two structs named `CallTrace`?

Reply:
One has serializable (copied from blockifier) structs inside them, one has blockifier structs. It will be removed in the future #1602

Reply:
Discussed offline, irrelevant